src/browser/api/system.js

```diff
@@ -304,6 +305,19 @@ exports.System = {
             return process.env[varsToExpand] || null;
         }
     },
+    getFocusedWindow: function() {
+        const fWin = electronBrowserWindow.getFocusedWindow();
+        if (fWin) {
```

Comment:
A space above would be nice

Reply:
do you mean a blank line above line 308?

Reply:
yeah